tests/all/module_serialize.rs

```diff
@@ -21,7 +21,7 @@ fn test_version_mismatch() -> Result<()> {
         Ok(_) => bail!("expected deserialization to fail"),
         Err(e) => assert_eq!(
             e.to_string(),
-            "Module was compiled with incompatible Wasmtime version 'x.25.0'"
+            "Module was compiled with incompatible Wasmtime version 'x.26.0'"
         ),
```

Comment:
Perhaps we should just leave the version number off the assertion so it's not a maintenance issue on version bumps?

Reply:
Good point -- now it's just `.starts_with("Module was compiled with incompatible Wasmtime version")`. Thanks!

Reply:
The maintenance issue could be prevented by overwriting the whole version number instead of just the first part I think. Keeping the version number could still be useful.

Reply:
That gets a little more complex, I think, especially when the version string gets longer (0.100.0)... this is a little too in-the-weeds IMHO for a simple "does the serialization check the version" test; any change in the version should suffice and any error message that starts with "Module was compiled with incompatible Wasmtime version" is certainly the right error that we want to see. If @peterhuene wants to make the test fancier in the future then we can certainly do that but I don't think we should hold the release for that.